CHANGES

```diff
@@ -81,6 +97,9 @@ Standard Library
 
 Plugins
 
+- The Ssreflect plugin is now distributed with Coq. Its documentation has
+  been integrated as a chapter of the reference manual. This chapter is
+  still imperfect so feedback is welcome.
```

Comment:
That has the same flavor as "imperfect", to me. Actually, I'd say "preliminary", and omit "still". 

Or how about "work in progress"?

But not so important, in any case.